src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 CHANGELOG
 =========
 
+5.3
+---
+
+ * Add a `valuesFormat` option to comparison constraints to format datetimes in
+   placeholders:
+   ```php
+   /**
+    * @Assert\EqualTo(valuesFormat="Y-d-m", value="2020-01-01")
```

Comment:
I would move valuesFormat to the second position

Reply:
Done